src/crypto/DeviceList.js

```diff
@@ -318,6 +318,10 @@ export default class DeviceList {
                 this._sessionStore.storeEndToEndDevicesForUser(
                     userId, storage,
                 );
+
+                if (token) {
+                    this._sessionStore.storeEndToEndDeviceSyncToken(token);
```

Comment:
hm. ooi, is there any reason why this would differ from the general /sync token? presumably this will collide with @kegsay's incremental sync stuff?